src/form-elements/FormElementCheckBoxes.tsx

```diff
@@ -88,7 +88,9 @@ function FormElementCheckboxes({
               options={filteredOptions}
               selected={selectedValues}
               disabled={element.readOnly}
-              onChange={onChange}
+              onChange={(element, value) => {
+                onChange(element, value)
```

Comment:
?

Reply:
ah yeah the next line got removed so I'll set that back